src/pages/catalog-page/catalog/filtering/options/category-options/handle-category-change.ts

```diff
@@ -0,0 +1,25 @@
+import type { NavigateFunction } from 'react-router-dom';
+import { catalogStore } from '../../../../../../store/catalog-store';
+import { PagePath } from '../../../../../../router/enums';
+import { messages } from './messages';
+
+export const handleCategoryChange = (
+  event: React.ChangeEvent<HTMLSelectElement>,
+  navigate: NavigateFunction
+) => {
+  const selectedId = event.target.value;
+  if (selectedId === messages.defaultTitle) {
```

Comment:
тож самое. просто как пустую строку обрабатывай
```suggestion
  if (!selectedId) {
```